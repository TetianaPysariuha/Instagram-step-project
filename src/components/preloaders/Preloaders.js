import styles from './Preloaders.module.scss';

function Preloaders() {
  return (
    <>
      <span className={styles.backgroundAnimation} />
      <div className={styles.loader} />
    </>
  );
}

export default Preloaders;
