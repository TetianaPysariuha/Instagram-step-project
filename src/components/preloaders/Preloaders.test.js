import { render } from '@testing-library/react';
import Preloaders from './Preloaders';

describe('Preloaders renders', () => {
  test('Should Preloaders render', () => {
    const { asFragment } = render(<Preloaders />);
    expect(asFragment()).toMatchSnapshot();
  });
});
