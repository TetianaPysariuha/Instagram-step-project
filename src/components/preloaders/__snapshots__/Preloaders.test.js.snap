// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preloaders renders Should Preloaders render 1`] = `
<DocumentFragment>
  <span
    class="backgroundAnimation"
  />
  <div
    class="loader"
  />
</DocumentFragment>
`;
